### PR TITLE
fix: Adjust Galleries for You header

### PR DESCRIPTION
**Pull request by olerichter00**
### Description

Adjust Galleries for You header.

| | |
| --- | --- |
|Simulator Screen Shot - iPhone 14 Pro - 2023-06-15 at 13 09 34 |Simulator Screen Shot - iPhone 14 Pro - 2023-06-15 at 12 42 16 |

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Scenes/GalleriesForYou/GalleriesForYouScreen.tsx [hunk 1]
```diff
@@ -158,7 +175,7 @@ const GalleriesForYouPlaceholder: React.FC = () => {
       <Flex testID="PlaceholderGrid">
         <GalleriesForYouHeader />
 
-        <Flex px={2} mt={1} mx="auto">
+        <Flex px={2} mx="auto">
```
Comment (MounirDhahri) [inline src/app/Scenes/GalleriesForYou/GalleriesForYouScreen.tsx:178]: **Q:** is `mx="auto"`

File: src/app/Scenes/GalleriesForYou/GalleriesForYouScreen.tsx [hunk 2]
```diff
@@ -36,6 +37,16 @@ export const GalleriesForYou: React.FC<GalleriesForYouProps> = ({ location }) =>
 
   const partners = extractNodes(data.partnersConnection)
 
+  const { headerElement, scrollProps } = useStickyScrollHeader({
+    header: (
+      <Flex flex={1} pl={6} pr={4} pt={0.5} flexDirection="row">
```
Comment (MounirDhahri) [inline src/app/Scenes/GalleriesForYou/GalleriesForYouScreen.tsx:42]:
```suggestion
      <Flex flex={1} pl={6} pr={4} pt={0.5} >
```

**Review by MounirDhahri — DISMISSED** [review]
LGTM

File: src/app/Scenes/GalleriesForYou/GalleriesForYouScreen.tsx (hunk 2 above)
Comment (olerichter00) [reply src/app/Scenes/GalleriesForYou/GalleriesForYouScreen.tsx:42]: 👍

**Pull request merged**